packages/paste-core/components/form/src/FormInput.tsx

```diff
@@ -81,7 +80,7 @@ const FormInput = React.forwardRef<HTMLInputElement, FormInputProps>(
     }
 
     return (
-      <FieldWrapper readOnly={readOnly} disabled={disabled} hasError={hasError}>
+      <FieldWrapper {...typeProps} readOnly={readOnly} disabled={disabled} hasError={hasError}>
```

Comment:
I would favour just passing down `type` here. If it's ever a number, `typeProps` is an object, and not a string and that feels weirdly misleading for FieldWrapper.

Reply:
Passing `type` now.